src/utils/CodecParser.h

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright (C) 2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include <bento4/Ap4Ac3Parser.h>
+#include <bento4/Ap4AdtsParser.h>
+#include <bento4/Ap4Eac3Parser.h>
+
+#define AP4_ADTS_HEADER_SIZE 7
+#define AP4_ADTS_SYNC_MASK 0xFFF6
+#define AP4_ADTS_SYNC_PATTERN 0xFFF0
+#define AP4_AC3_SYNC_PATTERN 0x0B77
+#define AP4_AC4_SYNC_MASK 0xFFF0
+#define AP4_AC4_SYNC_PATTERN 0x0AC40

```

Comment:
new code should not use defines, but for example
`constexpr AP4_UI32 AP4_ADTS_SYNC_PATTERN = 0xFFF0;`